yarn.lock

```diff
@@ -6300,17 +6300,17 @@ __metadata:
   languageName: node
   linkType: hard
 
-"hugo-bin@npm:0.86.0":
-  version: 0.86.0
-  resolution: "hugo-bin@npm:0.86.0"
+"hugo-bin@npm:0.89.0":
```

Comment:
Your PR description says 0.101.0 but this says 0.89.0.... was this on purpose?

Reply:
Yes that is correct 0.89.0 is the hugo-bin version but it actually means using hugo 0.101.0. 
Its a bit confusing because hugo-bin is using its own versioning you can see what i mean here.
https://github.com/fenneclab/hugo-bin/blob/main/package.json#L3-L4